R/map_shapefile.R

```diff
@@ -0,0 +1,90 @@
+#' Function for making basic choropleth maps using the tigris package and ggplot
+#'
+#' @param voter_file a voter file with latitude, longitude, or latitude/longitude and other geographic data.
+#' @param shape_file a shape file based on desired ecologicla unit (i.e. state, county, block, tract)
+#' @param crs the Coordinate reference system, default is crs="+proj=latlong +ellps=GRS80 +no_defs"
+#' @param title the tile of the map
+#'
+#' @export shape_file
+#' @export shape_list
+#'
+#' @return Plots of mapped ecological units desired and voter latitude and longitudes
+#'
+#' @importFrom sf
+#' @importFrom ggplot2
+#' @importFrom ggmap
```

Comment:
I think these three lines need to be "@import <package" instead of importFrom, or else checks won't pass.